lib/ts/fetch.ts

```diff
@@ -254,6 +254,21 @@ export default class AuthHttpRequest {
             logDebugMessage("doRequest: Returning without interception");
             return await httpCall(config);
         }
+
+        const origHeaders = new Headers(
+            config !== undefined && config.headers !== undefined ? config.headers : url.headers
+        );
+
+        if (origHeaders.has("Authorization")) {
```

Comment:
what if the user has set lower case authorization?

Reply:
the Headers class handles these as case insensitive.